packages/fluentui/docs/src/examples/components/Dropdown/Performance/DropdownManyItems.perf.tsx

```diff
@@ -10,6 +10,7 @@ const inputItems = _.times(300, (i: number) => ({
 
 const DropdownManyItemsPerf = () => <Dropdown defaultOpen items={inputItems} placeholder="Select your hero" />;
 
+DropdownManyItemsPerf.iterations = 1000;
```

Comment:
I forgot that defaultIterations for Fluent perf tests is 1. This test is already rendering 300 items per iteration (line 5 above), so the most iterations we should have is 3 or 5.

Reply:
done in the latest commit, thank you!